### PR TITLE
merchstore improvements

### PR DESCRIPTION
**Pull request by jlevine18**

### CODE REVIEW

**Comment by jlevine18** [comment]
Don't merge this yet, adding other improvements (description support, capacity alerts)

File: src/app/(membership)/merch/page.tsx [hunk 1]
```diff
@@ -188,6 +212,7 @@ const MerchItem = () => {
                 label="Size"
                 placeholder="Select an size"
                 selectedKeys={[size]}
+                disabledKeys={merchList["sizes"].filter(filterSoldOut)}
```
Comment (devksingh4) [inline src/app/(membership)/merch/page.tsx:215]: We should also add a textual indicator (e.g. "Sold out") in addition to disabling the key.

**Review by devksingh4 — CHANGES_REQUESTED** [review]
See review comments. I would also like to disable the "purchase now" button if no size is selected, or if all sizes are sold out. Otherwise, looks good great work!

File: src/app/(membership)/merch/page.tsx (hunk 1 above)
Comment (jlevine18) [reply src/app/(membership)/merch/page.tsx:215]: changed

**Review by devksingh4 — APPROVED** [review]
lgtm

**Pull request merged**